backend/src/scripts/migrateV2.ts

```diff
@@ -153,6 +174,169 @@ async function migrateModel(modelId: string) {
+      const path = version.files.rawCodePath
+      const size = await getObjectContentLength(bucket, path)
+      if (size) {
+        const v2File = await FileModel.findOneAndUpdate(
+          { modelId, bucket, path },
+          {
+            modelId,
+            name: `${version.version}-rawCodePath.zip`,
+            mime: 'application/x-zip-compressed',
+            bucket,
+            size,
+            path,
+            complete: true,
+
+            createdAt: version.createdAt,
+            updatedAt: version.updatedAt,
+          },
+          {
+            new: true,
+            upsert: true, // Make this update into an upsert
+            timestamps: false,
+          },
+        )
+        v2Files.push(v2File._id.toString())
+      }
+    }
+    if (version.files.rawDockerPath) {
+      const path = version.files.rawDockerPath
+      const size = await getObjectContentLength(bucket, path)
+      if (size) {
+        const v2File = await FileModel.findOneAndUpdate(
+          { modelId, bucket, path },
+          {
+            modelId,
+            name: `${version.version}-rawDockerPath.tar`,
+            mime: 'application/octet-stream',
+            bucket,
+            size,
+            path,
+            complete: true,
+
+            createdAt: version.createdAt,
+            updatedAt: version.updatedAt,
+          },
+          {
+            new: true,
+            upsert: true, // Make this update into an upsert
+            timestamps: false,
+          },
+        )
+        v2Files.push(v2File._id.toString())
+      }
+    }
+
+    const semver = `0.0.${i + 1}`
+    await Release.findOneAndUpdate(
+      { modelId, semver },
+      {
+        modelId,
+        modelCardVersion: i,
+
+        semver,
+        notes: `Migrated from V1. Orginal version ID: ${version.version}`,
+
+        minor: false,
+        draft: false,
+
+        fileIds: v2Files,
+        // Not sure about images
+        images: [],
+
+        createdBy: 'system',
+        createdAt: version.createdAt,
+        updatedAt: version.updatedAt,
+      },
+
+      {
+        new: true,
+        upsert: true, // Make this update into an upsert
+        timestamps: false,
+      },
+    )
+
+    const versionApprovals = await ApprovalModel.find({
+      version: version._id,
+      approvalCategory: ApprovalCategory.Upload,
+    }).sort({ createdAt: 1 })
+
+    for (const approval of versionApprovals) {
+      let role
+      if (approval.approvalType === ApprovalTypes.Manager) {
+        role = 'msro'
+      } else if (approval.approvalType === ApprovalTypes.Reviewer) {
+        role = 'mtr'
+      }
+
+      const responses: ReviewResponse[] = []
+      for (let i = 0; i < approval.approvers.length; i++) {
+        const approver = approval.approvers[i]
+        if (approval.status === ApprovalStates.Accepted) {
+          responses.push({
+            user: toEntity('user', approver.id),
+            decision: Decision.Approve,
```

Comment:
Add a comment to both of these responses to note that the response has been migrated from V1